lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -91,8 +92,10 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
     @Input()
     set tags(tags: string[]) {
         this._tags = [...tags];
+        this.tagsToDisplay = this._tags.map((tag) => {
```

Comment:
Can we use the getter for tags here? If some field has getter then we should refer to it by getter.

Reply:
not addressed, I meant here something like: 
this.tags.map instead of this._tags.map, can you just remove this _ character as we have getter which we can use?

Reply:
ah sorry @tamaragruszka, I see you fixed this for that file but the same thing is in different file, here: 
<img width="621" alt="image" src="https://github.com/user-attachments/assets/c22beee0-aaa3-4aaf-adbd-3d4bd7261e13">

Can you fix?

Reply:
it's lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts file